lighthouse-core/config/default.js

```diff
@@ -12,8 +12,17 @@ module.exports = {
       "viewport-dimensions",
       "theme-color",
       "manifest",
+      "chrome-console-messages",
       "image-usage",
-      "accessibility"
+      "accessibility",
+      "dobetterweb/all-event-listeners",
+      "dobetterweb/anchors-with-no-rel-noopener",
+      "dobetterweb/appcache",
+      "dobetterweb/domstats",
+      "dobetterweb/optimized-images",
```

Comment:
sg. do  you want to remove the network.enable/disable stuff in that gatherer now, then?